src/gym_electric_motor/physical_systems/electric_motors/externally_excited_synchronous_motor.py

```diff
@@ -92,48 +93,83 @@ class ExternallyExcitedSynchronousMotor(SynchronousMotor):
-            [                     0,                                           0, -mp['r_s'],                                           0,                                0,   1,                                0, -mp['l_d'] * mp['p'],                                                      0, -mp['p'] * mp['l_m']],
-            [                     0, mp['l_m'] * mp['r_s'] / (sigma * mp['l_d']),          0,                          -mp['r_e'] / sigma, -mp['l_m'] / (sigma * mp['l_d']),   0,                        1 / sigma,                    0, -mp['p'] * mp['l_m'] * mp['l_q'] / (sigma * mp['l_d']),                    0],
-            [               mp['p'],                                           0,          0,                                           0,                                0,   0,                                0,                    0,                                                      0,                    0],
-        ])
-
-        self._model_constants[self.I_SD_IDX] = self._model_constants[self.I_SD_IDX] / mp['l_d']
-        self._model_constants[self.I_SQ_IDX] = self._model_constants[self.I_SQ_IDX] / mp['l_q']
-        self._model_constants[self.I_E_IDX] = self._model_constants[self.I_E_IDX] / mp['l_e']
+        sigma = 1 - mp["l_m"] ** 2 / (mp["l_d"] * mp["l_e"])
+        self._model_constants = np.array(
+            [
+                #                 omega,                                         i_d,        i_q,                                         i_e,                              u_d, u_q,                              u_e,          omega * i_d,                                            omega * i_q,           omega * i_e
+                [
+                    0,
+                    -mp["r_s"] / sigma,
+                    0,
+                    mp["l_m"] * mp["r_e"] / (sigma * mp["l_e"]),
+                    1 / sigma,
+                    0,
+                    -mp["l_m"] / (sigma * mp["l_e"]),
+                    0,
+                    mp["l_q"] * mp["p"] / sigma,
+                    0,
+                ],
+                [
+                    0,
+                    0,
+                    -mp["r_s"],
+                    0,
+                    0,
+                    1,
+                    0,
+                    -mp["l_d"] * mp["p"],
+                    0,
+                    -mp["p"] * mp["l_m"],
+                ],
+                [
+                    0,
+                    mp["l_m"] * mp["r_s"] / (sigma * mp["l_d"]),
+                    0,
+                    -mp["r_e"] / sigma,
+                    -mp["l_m"] / (sigma * mp["l_d"]),
+                    0,
+                    1 / sigma,
+                    0,
+                    -mp["p"] * mp["l_m"] * mp["l_q"] / (sigma * mp["l_d"]),
+                    0,
+                ],
+                [mp["p"], 0, 0, 0, 0, 0, 0, 0, 0, 0],
```

Comment:
On the one hand, I get that the formatter prefers this shape of code.
On the other hand, the matrix calculations involved have been more intuitive the way it was before refactoring. 
I guess this needs further discussion.

(Applies to several files)

Reply:
Yea, formating should definitely be disabled for this kind of calucations.
Leaving this here for future reference:

```
# fmt: off

def my_unformatted_function():
   do_something()

# fmt: on
```